Website/src/rendering/src/components/SpeakersGrid.tsx

```diff
@@ -1,38 +1,42 @@
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import Link from 'next/link';
-import { Text, Field, ImageField, Image } from '@sitecore-jss/sitecore-jss-nextjs';
+import { Text, Field, Image } from '@sitecore-jss/sitecore-jss-nextjs';
 import { ComponentProps } from 'lib/component-props';
 import { faStar } from '@fortawesome/free-solid-svg-icons';
 
 type Speaker = {
-  fields: {
-    Name: Field<string>;
-    Role: Field<string>;
-    Picture: ImageField;
-    Featured: Field<boolean>;
-  };
+  name: string;
+  picture: { jsonValue: { value: { src: string } } };
+  featured: Field<boolean>;
+  role: Field<string>;
 };
 
 type SpeakersGridProps = ComponentProps & {
   fields: {
-    items: Speaker[];
+    data: {
+      item: {
+        children: {
+          results: Speaker[];
+        };
+      };
+    };
   };
 };
 
 const SpeakersGrid = (props: SpeakersGridProps): JSX.Element => (
   <div className="section__speakers__grid container">
-    {props.fields.items &&
-      props.fields.items.map((speaker, index) => (
-        <Link key={index} href={'/speakers/' + speaker.fields.Name.value} passHref>
+    {props.fields.data.item.children.results &&
```

Comment:
Everywhere we check for the presence of a value before outputting it or mapping it, we could change the code to use the `?.` operator. That way, we would have only one line for checks and one line for usage.
For example here: `props.fields?.data?.item?.children?.results && props.fields.data.item.children.results.map(...)`

Another example. Instead of:
```
{session.day &&
  session.day.targetItems &&
  session.day.targetItems.length > 0 &&
  session.day.targetItems.map (...)
```
We could just do:
```
{session.day?.targetItems &&
  session.day.targetItems.length > 0 &&
  session.day.targetItems.map (...)
```